explorer/src/Pos/Explorer/Socket/Methods.hs

```diff
@@ -338,8 +339,8 @@ addressSetByTxs tx txs =
     S.fromList $ txOutAddress <$> txs'
 
 getBlockTxs
-    :: forall ssc ctx m . ExplorerMode ctx m
-    => Block ssc -> m [TxInternal]
+    :: forall ssc ctx m . (ExplorerMode ctx m, ssc ~ SscGodTossing)
```

Comment:
Why is `ssc` parameter needed here at all?

Reply:
It's likely not needed but I tried to avoid removing `ssc` from more places than I wanted to (`Block` and `BlockHeader`) because it would've increased the size of this PR (e.g. if you remove `ssc` from a `forall`, you also need to change all type applications when there are type applications)